samples/boards/cc3220sf_launchxl/echo/README.rst

```diff
@@ -0,0 +1,55 @@
+.. _echo_sample:
```

Comment:
I'd recommend giving this a more specific name, such as 

    .. _wifi-echo-sample

Reply:
done.